packages/geo/package.json

```
@@ -56,7 +65,11 @@
 		"@aws-amplify/core": "^6.0.0"
 	},
 	"devDependencies": {
-		"@aws-amplify/core": "6.0.0"
+		"@aws-amplify/core": "6.0.0",
+		"typescript": "5.0.2"
+	},
+	"resolutions": {
```

Why do we need this resolution? We should try to use the `^2.5.0` in the deps to improve flattening across the lib. If not possible for some reason, I would just update the dependency for Geo instead of using a resolution.

```suggestion
	"resolutions": {
```

---

I don't recall the reason now, will look into it